fluss-server/src/main/java/org/apache/fluss/server/kv/KvManager.java

```diff
@@ -304,6 +312,9 @@ public void dropKv(TableBucket tableBucket) {
         if (dropKvTablet != null) {
             TablePath tablePath = dropKvTablet.getTablePath();
             try {
+                // Unregister RocksDB metrics from TableMetricGroup
+                serverMetricGroup.unregisterRocksDBMetrics(tablePath, tableBucket);
```

Comment:
We already unregister RocksDB metrics in `ReplicaManager#stopReplica`, which invokes `TableMetricGroup#removeBucketMetricGroup(...)` by `TabletServerMetricGroup#removeTableBucketMetricGroup`. That method, in turn, handles the unregistration of the RocksDB metrics.